### PR TITLE
Hide brand menu when not authorized

### PR DESCRIPTION
**Pull request by MrBurrBurr**

### CODE REVIEW

File: src/components/NavigationBrand.vue [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 		<span class="brand__name brand__name--small"><b>A</b>SF</span>
 		<span class="brand__name brand__name--big"><b>Archi</b>SteamFarm</span>
-		<div class="brand__icon">
+		<div v-if="validPassword" class="brand__icon">
 			<font-awesome-icon v-if="brandMenu" icon="times"></font-awesome-icon>
 			<font-awesome-icon v-else icon="angle-down"></font-awesome-icon>
 		</div>
 
-		<transition name="brand__menu">
+		<transition v-if="validPassword" name="brand__menu">
```
Comment (Aareksio) [inline src/components/NavigationBrand.vue]: Do not apply `v-if` on transition, instead add the condition to transitioned element itself

**Comment by MrBurrBurr** [comment]
Is there an easy way to change css with `v-if`? I would like to remove `cursor: pointer;` if not authorized.

**Comment by Aareksio** [comment]
Not with `v-if`, but you can use `v-bind` syntax to change `style` or `class` dynamically to achieve the effect you need, ex:
```Vue
<div 
	class="brand" 
	:class="{ 'brand--not-logged': !validPassword }" 
	:style="{ cursor: validPassword ? 'pointer' : 'initial' }">
</div>
```

I'd honestly prefer to add `.app--not-logged` to `.app` and then utilize SCSS nesting:
```SCSS
.brand {
	// ...

	.app--not-logged & {
		cursor: initial;
	}
}
```

File: src/components/NavigationBrand.vue [hunk 2]
```diff
@@ -1,14 +1,14 @@
 <template>
-	<div class="brand" @click="toggleBrandMenu">
+	<div class="brand" :class="{ 'app--not-authorized': !validPassword }" @click="toggleBrandMenu">
```
Comment (Aareksio) [inline src/components/NavigationBrand.vue]: I don't think it's needed here ;)

**Pull request merged**